zkg

```diff
@@ -37,6 +37,19 @@ if os.path.isdir(ZEEK_PYTHON_DIR):
 else:
     ZEEK_PYTHON_DIR = None
 
+# Similarly, make Zeek's binary installation path available by
+# default. This helps package installations succeed that require
+# e.g. zeek-config for their build process.
+ZEEK_BIN_DIR = '@ZEEK_BIN_DIR@'
+if os.path.isdir(ZEEK_BIN_DIR):
+    try:
+        if ZEEK_BIN_DIR not in os.environ['PATH']:
+            os.environ['PATH'] += os.pathsep + ZEEK_BIN_DIR
```

Comment:
Would prepending better guarantee matching against the associated Zeek install ?

```suggestion
            os.environ['PATH'] = ZEEK_BIN_DIR + os.pathsep + os.environ['PATH']
```

Reply:
Yeah I thought about that one too and wasn't sure. Johanna's problem was just that `zeek-config` wasn't found at all. Looking at it, I like it. It makes things more predictable. Thanks!